api/candidatesApiClient.ts

```diff
@@ -7,4 +7,15 @@
     //put into list
     
     return ["Michael Walker", "Stephen Jackson"];
+}
+
+const CREATE_CANDIDATE_API_URL = "https://testswitch-api-staging.herokuapp.com/candidates/create"; 
+
+export async function tryCreateApi(userId: string, password: string, email: string): Promise<number>{
+    //TODO: delete test constant and return actual status code from api
+    // const response = await fetch(API_URL); etc
+    // const statusCode = response.status;
+
+    //Check how the page works with 200, 403, and any other error code.
+    return 200;
```

Comment:
so this won't (easily) work

Our API is currently build to accept form data (rather than JSON etc)
But this method looks like its going to want to submit some JSON.

I think we should really either
- change the API to accept JSON data (ie `[FromBody]` rather than `[FromForm]`
- change the frontend to submit the form rather than submitting via JS. 

I slightly prefer changing the frontend, because then the frontend would actually work without JS, which is nice! :)